web/themes/custom/accessibility_theme/src/patterns/components/media-item/media-item.twig

```diff
@@ -0,0 +1,29 @@
+{{ attach_library('accessibility_theme/media-item') }}
+
+<div class="media-item {{ modifier }}" {{ extra_attributes }}>
```

Comment:
Let's use the method for attributes: https://mariohernandez.gitbook.io/components/chapter-3/drupal-attributes